SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -113,6 +113,9 @@ void USpatialSender::Init(USpatialNetDriver* InNetDriver, FTimerManager* InTimer
 	{
 		PeriodicallyProcessOutgoingRPCs();
 	}
+
+	// TODO: make it configurable in the future
+	MaxInitialOnlyRequestPerTick = MAX_INITIAL_ONLY_REQUEST_PER_TICK;
```

Comment:
Was there any specific performance concerns around adding this upper limit?

Reply:
Just to prevent the runtime from overloaded if excessive InitialOnly requests need to be sent to runtime, so far as I know in Umi's scenario if the player enters the scene, it would dynamically load around 1000 blast actors